### PR TITLE
fixing parser logic issue

### PR DESCRIPTION
**Pull request by tommorris**
I made a mistake earlier and broke the parser logic. This fixes it.

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -23,11 +23,12 @@ def test_parse_another_description():
     item_names = [x["title"] for x in result["items"]]
     assert u'Style' in item_names
     assert u'Care and composition' in item_names
+    import pprint
```
Comment (jpluscplusm) [inline tests/test.py]: Silly question: should this line be *here*? Should it be present at all?

Comment (tommorris) [inline tests/test.py]: @jpluscplusm: Fixed.

**Comment by jpluscplusm** [comment]
This is one for @norm to merge, given its scope.

**Comment by tommorris** [comment]
Thanks, @norm.

**Pull request merged**